libraries/core-react/src/List/List.tsx

```diff
@@ -41,6 +41,6 @@ const List = forwardRef<HTMLUListElement | HTMLOListElement, Props>(
   },
 )
 
-// List.displayName = 'List'
+List.displayName = 'List'
```

Comment:
Is `displayName` needed again?

Reply:
Hmm, no, I was just investigating another bug. 👍